pkg/3scale/amp/operator/system_mysql_options_provider_test.go

```diff
@@ -91,9 +92,9 @@ func defaultSystemMysqlOptions(opts *component.SystemMysqlOptions) *component.Sy
 
 func TestGetMysqlOptionsProvider(t *testing.T) {
 	tmpFalseValue := false
-	systemMysqlRootPassword := "rootPassw1"
-	systemMysqlDatabaseName := "myDatabaseName"
-	databaseURL := fmt.Sprintf("mysql2://root:%s@system-mysql/%s", systemMysqlRootPassword, systemMysqlDatabaseName)
+	//systemMysqlRootPassword := "rootPassw1"
+	//systemMysqlDatabaseName := "myDatabaseName"
```

Comment:
```suggestion
```

Should these commented out lines be removed?

Reply:
updated/removed. Thank you